### ISSUE TITLE
[Feature Request] Make the USER env var configurable

### ISSUE THREAD

**Issue by Gerifield** [asciinema/asciinema#362]
Hi,
I use asciinema from different machines with different USER env vars.
I didn't want to permanently change the variable on my other machine and it would be great if this variable could come from the config file too.

This is the related line I guess: https://github.com/asciinema/asciinema/blob/develop/asciinema/commands/command.py#L10

**Comment by sickill** [comment]
You can always make a shell alias like this:

`alias asciinema='env USER=foo asciinema'`

Would that work?

**Comment by Gerifield** [comment]
Yes, but all the other (important) params could be set in the config file.
The other thing, this would also set the USER env var "inside" the asciinema command and if you'd like to have the original value in you env you should set it back to original inside, during the recording.

**Comment by sickill** [comment]
The `USER` env var is only meaningful when your install ID hasn't been registered on the server (asciinema.org or your own), and it's used for populating temporary username in the database, which we display for asciicasts recorded by non verified users. If you register all your machines with `asciinema auth` then this username should not matter at all, and would be ignored - the username you've setup on web for your account is then used for display.

The fact we always send `USER` as username path in basic auth header is to support the "unknown install ID" case. See here: https://github.com/asciinema/asciinema-server/blob/develop/lib/asciinema_web/controllers/api/asciicast_controller.ex#L71 and here https://github.com/asciinema/asciinema-server/blob/develop/lib/asciinema/accounts/accounts.ex#L141-L150

In other words, we might as well not send `USER` at all, and put `""` (blank) as basic auth username, in which case asciicasts for unverified users would display "user:123456" (where 123456 is user ID) as username on the website. Long time ago I thought it would be ok to use it, so we have something to display there, until user fully sets up the account.
